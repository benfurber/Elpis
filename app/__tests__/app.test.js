import "react-native";
import React from "react";
import { shallow } from "enzyme";

import App from "../index";

describe("App", () => {
  it("renders correctly", () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
  });
});
