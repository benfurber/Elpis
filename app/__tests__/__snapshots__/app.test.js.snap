// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Text
        style={
          Object {
            "fontSize": 20,
            "margin": 10,
            "textAlign": "center",
          }
        }
      >
        Welcome to Elpis!
      </Text>,
      "style": Object {
        "alignItems": "center",
        "backgroundColor": "#F5FCFF",
        "flex": 1,
        "justifyContent": "center",
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": "Welcome to Elpis!",
        "style": Object {
          "fontSize": 20,
          "margin": 10,
          "textAlign": "center",
        },
      },
      "ref": null,
      "rendered": "Welcome to Elpis!",
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Text
          style={
            Object {
              "fontSize": 20,
              "margin": 10,
              "textAlign": "center",
            }
          }
        >
          Welcome to Elpis!
        </Text>,
        "style": Object {
          "alignItems": "center",
          "backgroundColor": "#F5FCFF",
          "flex": 1,
          "justifyContent": "center",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Welcome to Elpis!",
          "style": Object {
            "fontSize": 20,
            "margin": 10,
            "textAlign": "center",
          },
        },
        "ref": null,
        "rendered": "Welcome to Elpis!",
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
